plausible/docker-compose.yml

```
@@ -0,0 +1,44 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: plausible_1
+      APP_PORT: 9091
```

This does not need to match the `port` key in the umbrel-app.yml. Instead this can just point to the default Plausible port of 8000.

```suggestion
      APP_PORT: 8000
```